### PR TITLE
(fix)The modal nav on each app

### PR DESCRIPTION
**Pull request by SOAP1914**
To let JAWS nav thru the app modal (top nav)

### CODE REVIEW

**Comment by SOAP1914** [comment]
@arw180 @blaino @clarkjefcoat @wski @hoganmaps

File: app/js/components/quickview/Header.jsx [hunk 1]
```diff
@@ -30,12 +30,12 @@ var QuickviewHeader = React.createClass({
         return (
             <div className="quickview-header">
                 <div className="quickview-header-info">
-                    <button type="button" className="close" aria-hidden="true" onClick={ this.props.onCancel }><i className="icon-cross-16-grayDark"></i></button>
```
Comment (blaino) [inline app/js/components/quickview/Header.jsx]: Why was aria-hidden ever "true"?

**Comment by blaino** [comment]
Aside from my not understanding how aria-hidden is supposed to be used, LGTM. :+1:

File: app/js/components/quickview/Header.jsx (hunk 1 above)
Comment (SOAP1914) [inline app/js/components/quickview/Header.jsx]: lol np good question, i'll add to best practice (thanks)
In Bootstrap Modals...
The purpose of the aria-hidden on the close button is that in that markup the "X" or "times"-symbol won't tell screen readers much.

To make it accessible the close button should look something like this:

<button type="button" title="Close">
  <span aria-hidden="true">&times;</span>
  <span class="hide">Close</span>
</button>

more info here:
http://getbootstrap.com/javascript/#modals

**Pull request merged**